src/Book.java

```diff
@@ -0,0 +1,33 @@
+class Book {
+    private String title;
+    private Author authorName;
```

Comment:
```suggestion
    private Author author;
```
это не название автора, а сам автор